docker-compose.yml

```diff
@@ -16,9 +16,11 @@ services:
       - RDS_PORT=5432
       - SOLR_URL=http://solr:8983/solr/dlme
       - SECRET_KEY_BASE
+      - REDIS_HOST
+      - REDIS_PORT
       # For development
-      - RAILS_SERVE_STATIC_FILES=true
-      - RAILS_ENV=development
+#      - RAILS_SERVE_STATIC_FILES=true
+#      - RAILS_ENV=development
```

Comment:
If we don't set development here, then it's running in production mode, right?  Why?

Reply:
Because otherwise the sidekiq config doesn't work. Would take some additional tweaking.

Reply:
Because `config/environments/development.rb` doesn't set a queue_adapter.